src/user/user.py

```diff
@@ -21,23 +26,31 @@ def is_admin(self):
     @property
     def games_played(self):
         # Query game.players
-        return 0
+        return []
 
     @property
     def games_won(self):
         # Query Game.winners
-        return None
+        return []
 
     @property
     def level(self):
         return 1
 
     @property
+    def rating(self):
+        if not self.rank_data:
+            self.rank_data = Rating()
+            self.put()
+        return trueskill.Rating(
+            mu=self.rank_data.points, sigma=self.rank_data.elasticity)
+
+    @property
     def win_percentage(self):
         # self.games_won / self.games_played
         games_played = self.games_played
         if games_played:
-            return self.games_won / self.games_played
+            return len(self.games_won) / len(self.games_played)
```

Comment:
If after a year each of us has played 1000 games, I don't think that this `@property` could take a long time to run. And I think `@properties` should not take a long time to run.